### ISSUE TITLE
Modification in @organisms/site/site-header/site-header.twig have no effect

### ISSUE THREAD

**Issue by sinyayadynya** [fourkitchens/emulsify#227]
In the new 2.3 version, I'm trying to add some class and/or role to .navigation in components/_patterns/03-organisms/site/site-header/site-header.twig as:

```
<header class="header">
  <div class="navigation" role="main">
    {{ content }}
  </div>
</header>
```

But this doesn't have any effect (cache cleared with 'drupal cr all').

Any idea why? Thanks for help.

**Comment by evanmwillhite** [comment]
Was this because of the competing installations as well @sinyayadynya?

**Comment by sinyayadynya** [comment]
@evanmwillhite Yes, the Emulsify theme was still installed while I was using a custom theme. Sorry for having missed that.
